bundles/binding/org.openhab.binding.plclogo/src/main/java/Moka7/IntByRef.java

```diff
@@ -0,0 +1,40 @@
+/*=============================================================================|
+|  PROJECT Moka7                                                         1.0.2 |
+|==============================================================================|
+|  Copyright (C) 2013, 2016 Davide Nardella                                    |
+|  All rights reserved.                                                        |

```

Comment:
@watou Ahh, I am unsure, what is correct way to dual-license sources? Do you know who can suggest right way?